# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/objc_video_capture/device_info.h
# Commit: 703360f0b861
# Full Hash: 703360f0b8611916dfb3724eda7f35345d3f4be7
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2022-12-13 04:11:09
# Regressor Bug: 1451394
# File Overlap Count: 1
# Description:
#   Bug 1451394 - Integrate with the libwebrtc camera backend for Mac. r=webrtc-reviewers,jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D163684
# ==============================================================================

diff -r 49002523b70d -r 703360f0b861 dom/media/systemservices/objc_video_capture/device_info.h
--- a/dom/media/systemservices/objc_video_capture/device_info.h	Mon Dec 12 15:47:22 2022 +0000
+++ b/dom/media/systemservices/objc_video_capture/device_info.h	Mon Dec 12 15:47:22 2022 +0000
@@ -18,8 +18,7 @@
 
 @class DeviceInfoIosObjC;
 
-namespace webrtc {
-namespace videocapturemodule {
+namespace webrtc::videocapturemodule {
 class DeviceInfoIos : public DeviceInfoImpl {
  public:
   DeviceInfoIos();
@@ -52,7 +51,6 @@
   DeviceInfoIosObjC* _captureInfo;
 };
 
-}  // namespace videocapturemodule
-}  // namespace webrtc
+}  // namespace webrtc::videocapturemodule
 
 #endif  // MODULES_VIDEO_CAPTURE_OBJC_DEVICE_INFO_H_